### PR TITLE
NGE-031: Add more path aliases and rules

### PR DESCRIPTION
**Pull request by JackBlakeston**
Add the following path aliases: @utils @hooks @context @const @types

Add eslint plugin that formats all possible import paths into path aliases as indicated in tsconfig

Add eslint plugin that sorts imports based on regexps

### CODE REVIEW

**Comment by JackBlakeston** [comment]
When this branch is reviewed, we must rebase main into it and lint all the code, as to avoid painful merge conflicts

File: packages/components/atoms/DisplayBox/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@no-gravity-elements/display-box",
-  "version": "1.0.6",
+  "version": "1.0.7",
```
Comment (AdrianRRo) [inline packages/components/atoms/DisplayBox/package.json]: Why do we increase the version of DisplayBox?

File: packages/components/atoms/InfoBox/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@no-gravity-elements/info-box",
-  "version": "1.0.6",
+  "version": "1.0.7",
```
Comment (AdrianRRo) [inline packages/components/atoms/InfoBox/package.json]: same here

**Review by AdrianRRo — COMMENTED** [review]
.

**Pull request merged**